spec/kitchen/transport/base_spec.rb

```diff
@@ -120,4 +120,16 @@
       connection.execute_with_retry("Hi", [123], 3, 1).must_equal "Hello"
     end
   end
+
+  describe "#retry?" do
+    it "raises Invalid Type Error for retry_on_exit_code" do
+      proc { connection.retry?(2, 2, "35 1", 35) }
+        .must_raise("undefined method `flatten' for \"35 1\":String")
+    end
+
+    it "when value is send in Array in Array" do
```

Comment:
This could be more clear written like, "returns true when the retryable exit codes are formatted in a nested array"